coordinator/internal/types/get_task.go

```diff
@@ -2,7 +2,8 @@ package types
 
 // GetTaskParameter for ProverTasks request parameter
 type GetTaskParameter struct {
-	ProverHeight int    `form:"prover_height" json:"prover_height"`
+	HardForkName string `form:"hard_fork_name" json:"hard_fork_name"`
```

Comment:
Umarshal will succeed if this is missing, right?

Reply:
Correct, Go json package treats all fields as optional.